changelog.md

```diff
@@ -2,15 +2,15 @@
 All notable changes to this project will be documented in this file.
 
 This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
-<br/>
+
```

Comment:
I wanted to include a bit of space to clearly visually separate releases. Multiple line breaks weren't doing the job.

We can try merging this in so you can see on [pymedphys.com](http://pymedphys.com). If it doesn't look nice, we can revert :)